MatrixKit/Categories/NSBundle+MatrixKit.m

```diff
@@ -15,16 +15,17 @@
  */
 
 #import "NSBundle+MatrixKit.h"
+#import "MXKViewController.h"
 
 @implementation NSBundle (MatrixKit)
 
 static NSString *customLocalizedStringTableName = nil;
 
 + (NSBundle*)mxk_assetsBundle
 {
-    NSString *bundleResourcePath = [NSBundle mainBundle].resourcePath;
-    NSString *assetPath = [bundleResourcePath stringByAppendingPathComponent:@"MatrixKitAssets.bundle"];
-    return [NSBundle bundleWithPath:assetPath];
+    NSBundle * bundle = [NSBundle bundleForClass:[MXKViewController class]];
```

Comment:
Just have a look at this closed issue here https://github.com/CocoaPods/CocoaPods/issues/3226
It says that when you are using use_frameworks in your podfile (this is mandatory in swift project), the pod is working as a Framework. The pod bundle is accessible by bundleForClass